.github/workflows/lint-check.yml

```diff
@@ -0,0 +1,22 @@
+# name: Lint Check
+
+# on:
+#   push:
+#     branches: ["main"]
+#   pull_request:
+
+# jobs:
+#   lint:
+#     runs-on: ubuntu-latest
+#     steps:
+#       # - name: Checkout code
+#       #   uses: actions/checkout@v4
+
+#       # - name: Set Up Python
+#       #   uses: actions/setup-python@v4
+#       #   with:
+#       #     python-version: '3.10'
+
+#       - name: Linting
+#         # TODO: uncomment above steps for checking out the code and setting up python when Linting is implemented.
+#         run: echo "black, isort, flake8, mypy still to come..."
```

Comment:
I'd just delete this, it's still in the git history if you need it later and it only adds noise right now.

Reply:
ok can do

Reply:
deleted [9e99969](https://github.com/Sindri-Labs/sindri-python/pull/14/commits/9e99969e1e146a8c44e5d51cb91897561b2254fa)